src/main/java/in/twizmwaz/cardinal/module/modules/wools/WoolObjective.java

```diff
@@ -145,11 +145,7 @@ public void onWoolPickup(InventoryClickEvent event) {
                         this.touched = true;
                         if (touchMessage) {
                             double newProx;
-                            if (location != null) {
-                                newProx = location.distance(place.getVector());
-                            } else {
-                                newProx = player.getLocation().toVector().distance(place.getVector());
-                            }
+                            newProx = player.getLocation().toVector().distance(place.getVector());
```

Comment:
@iEli2tyree011 Completely agree with you, but can't find a solution. Leaving it for you since you know more than me and understand the issue.